Fortran/exercise2/league_sim.f90

```diff
@@ -19,65 +22,9 @@ program football_league_simulator
  
     ! Print results
     call print_results(team_names, points)
-    
 
 contains
 
-subroutine simulate_match(team1, team2, points)
-    implicit none
-    integer, intent(in) :: team1, team2
-    integer, intent(inout) :: points(:)
-    
-    integer :: goals1, goals2
-    real :: g1, g2        
-
-    call random_seed()
-
-    ! Simulate goals for each team (max 5 goals)
-    call random_number(g1)
-    call random_number(g2)
-    goals1 = floor(6*g1) 
-    goals2 = floor(6*g2)
```

Comment:
Here the integer 6 is used, wheras in the module an int value `max_goals` is used with a value of 5. Suspect this change is unintentional.